src/app/account-settings/components/settings-security-password/settings-security-password.component.ts

```diff
@@ -70,12 +66,12 @@ export class SettingsSecurityPasswordComponent implements OnInit, OnDestroy {
           this.loading.emit(false)
 
           if (!value.password) {
-            this.errors = []
             this.form.reset()
             this.success = true
           } else {
-            this.success = false
-            this.errors = value.errors
+            this.form.controls['oldPassword'].setErrors({
```

Comment:
Is this error removed when the user updated the value of the password? 
Just asking to make sure this error is remove at some point